src/js/edu-benefits/components/NavButtons.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+
+// TODO update this when progress button moves
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+
+import { getSectionList } from '../utils/sections';
+
+export default class NavButtons extends React.Component {
```

Comment:
Can we not use the shared `ProgressButtons` for this instead of creating a whole new component?